### PR TITLE
[MOT]update window timer for pptracking

### PR DESCRIPTION
**Pull request by jerrywgz**

### CODE REVIEW

File: deploy/pptracking/python/mot_jde_infer.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def predict_video(detector, camera_id):
         os.makedirs(FLAGS.output_dir)
     out_path = os.path.join(FLAGS.output_dir, video_name)
     if not FLAGS.save_images:
-        fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+        fourcc = cv2.VideoWriter_fourcc(* 'mp4v')
```
Comment (nemonameless) [inline deploy/pptracking/python/mot_jde_infer.py]: *'mp4v'

File: deploy/pptracking/python/mot_sde_infer.py [hunk 2]
```diff
@@ -539,7 +539,7 @@ def predict_video(detector, reid_model, camera_id):
         os.makedirs(FLAGS.output_dir)
     out_path = os.path.join(FLAGS.output_dir, video_name)
     if not FLAGS.save_images:
-        fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+        fourcc = cv2.VideoWriter_fourcc(* 'mp4v')
```
Comment (nemonameless) [inline deploy/pptracking/python/mot_sde_infer.py]: *'mp4v'

File: deploy/pptracking/python/mot_jde_infer.py (hunk 1 above)
Comment (jerrywgz) [reply deploy/pptracking/python/mot_jde_infer.py]: done

File: deploy/pptracking/python/mot_sde_infer.py (hunk 2 above)
Comment (jerrywgz) [reply deploy/pptracking/python/mot_sde_infer.py]: done

**Review by zhiboniu — DISMISSED** [review]

**Review by nemonameless — APPROVED** [review]

**Review by zhiboniu — APPROVED** [review]

**Pull request merged**